R/zoop_synthesizer.R

```diff
@@ -537,7 +538,7 @@ Zoopsynther<-function(
     Zoop<-Zoop%>%
       dplyr::left_join(Lumpedkey%>%
                          dplyr::select("Taxlifestage", "Taxname_new", "SizeClass"),
-                       by=c("Taxlifestage", "SizeClass"))%>%
+                       by=c("Taxlifestage", "SizeClass"), relationship = "many-to-many")%>%
```

Comment:
Why was this change made?

Reply:
That was me trying to figure out why nothing was working. I've changed it back.